src/webapi/main.py

```diff
@@ -0,0 +1,37 @@
+from fastapi import FastAPI
+from pydantic import BaseModel
+
+
+class Bot(BaseModel):
+    name: str
+
+
+app = FastAPI(
+    title='Discord Bot Database',
+    docs_url='/'
+)
+
+
+@app.get('/api/bots')
+def get_bots():
+    return {'message': 'Hello World'}
+
+
+@app.get('/api/bots/{id}')
+def get_bot(id: int):
+    return {'message': 'Hello World'}
+
+
+@app.post('/api/bots')
+def post_bots(bot: Bot):
+    return {'message': 'Hello World'}
+
+
+@app.put('/api/bots/{id}')
+def put_bots(id: int, bot: Bot):
+    return {'message': 'Hello World'}
+
+
+@app.delete('/api/bots/{id}')
```

Comment:
```suggestion
@app.delete('/api/bots/{id}', status_code=204)
```
削除されたら 204 を返すべきだと思います！

Reply:
> **204 No Content**
>     **内容なし。** リクエストを受理したが、返すべきレスポンスエンティティが存在しない場合に返される。
>     例: POSTメソッドでフォームの内容を送信したが、ウェブブラウザの画面を更新しない場合に返される。

返すモノがないのでよさそう